doc/api/child_process.md

```diff
@@ -212,6 +212,9 @@ child runs longer than `timeout` milliseconds.
 *Note: Unlike the exec(3) POSIX system call, `child_process.exec()` does not
 replace the existing process and uses a shell to execute the command.*
 
+If this method is invoked as its [`util.promisify()`][]ed version, it returns
+a Promise for an object with `stdout` and `stderr` properties.
+
```

Comment:
I understand that it is quite tedious to do so, but examples would be good for these